templates/documents/general_view.html

```diff
@@ -309,8 +309,43 @@ <h4>{xl t='Contents'|escape:'html'}</h4>
-			{elseif $file->get_ccr_type($file->get_id()) ne "CCR" and
-       $file->get_ccr_type($file->get_id()) ne "CCD"}
+            {elseif $file->get_mimetype() eq "application/dicom"}
+            <!-- DWV -->
+            <div id="dwv" src="{$web_path}as_file=false">
+                <!-- Toolbar -->
+                <div class="toolbar"></div>
+                <div class="warn_diagnostic">Not For Diagnostic Use</div>
+                <!-- Layer Container -->
+                <div class="layerContainer">
+                    <canvas class="imageLayer">Only for HTML5 compatible browsers...</canvas>
+                </div><!-- /layerContainer -->
+            </div><!-- /dwv -->
+            <!-- Third party (dwv) -->
+            <script type="text/javascript" src="{$assets_static_relative}/modernizr-3-5-0/dist/modernizr-build.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-9-0-1/i18next.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-9-0-1/i18next.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-xhr-backend-1-4-3/i18nextXHRBackend.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/i18next-browser-languagedetector-2-0-0/i18nextBrowserLanguageDetector.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/konva-1-6-8/konva.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/magic-wand-js/js/magic-wand-min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/jszip-3-1-5/dist/jszip.min.js"></script>
+            <!-- Third party (viewer) -->
+            <script type="text/javascript" src="{$assets_static_relative}/jquery-ui-1-12-1/jquery-ui.min.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/flot-0-8-3/jquery.flot.js"></script>
+            <!-- decoders -->
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-0-21-0/decoders/pdfjs/jpx.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-0-21-0/decoders/pdfjs/util.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-0-21-0/decoders/pdfjs/arithmetic_decoder.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-0-21-0/decoders/pdfjs/jpg.js"></script>
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-0-21-0/decoders/rii-mango/lossless-min.js"></script>
+            <!-- Local (dwv) -->
+            <script type="text/javascript" src="{$assets_static_relative}/dwv-0-21-0/dist/dwv.min.js"></script>
+            <!-- i18n dwv wrapper -->
+            <script type="text/javascript" src="{$webroot}/library/js/dwv/dwv_i18n.js"></script>
```

Comment:
To confirm, it looks like this script does not come with the standard package, correct?
Should be able to do away with the locale scripts though, by setting the path at the bottom of the dwv_i18n.js to the locale files in the standard package in public/assets

Reply:
@bradymiller 

Yes indeed you are right. In fact I've been looking at more example dwv code to extend the viewer and it seems almost all of the code from the utils folder (https://github.com/ivmartel/dwv/tree/master/src/utils) will need to be pulled into OpenEMR somehow. None of these utility functions are included in the official distribution dwv js file. 

As for the locales, thanks for telling me :) - it is an oversight and will be fixed shortly.

Reply:
Thats ok on the utils stuff. Not much and looks like those scripts sometimes require mods etc. It's well worth having a dicom viewer :)

Reply:
When locale stuff is dealt with, then I think this is ready to bring into the main codebase.

Reply:
@bradymiller 

Thanks, I will take care of this tomorrow. Good to hear Firefox issue was solved but it would be nice to have better cross-browser compatibility so I will keep investigating.